packages/math-input/src/components/keypad/utils.ts

```diff
@@ -1,6 +1,8 @@
 import Keys from "../../data/key-configs";
 import {CursorContext} from "../input/cursor-contexts";
 
+export const expandedViewThreshold = 500;
```

Comment:
Is this simply based on heuristics or some calculation? If a calculation, do you mind leaving a comment?

Reply:
It's a magic number; I added a comment, but I'm not sure how to make it non-magical.